Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1070,11 +1070,12 @@ def compile(self, more, haveInstructions, glyfTable):
 				data = data + struct.pack(">HH", self.firstPt, self.secondPt)
 				flags = flags | ARG_1_AND_2_ARE_WORDS
 		else:
+			x, y = self.roundOffset()
```

Comment:
I see. But again that is what is happening already with coordinates. We may think to revise that too, not sure... /cc @behdad @justvanrossum